doc/manual/Configuration.md

```diff
@@ -843,3 +852,4 @@ must use the global option **dbus daemon** to point to the path, e.g. for
 Solaris with Tracker from OpenCSW:
 
     dbus daemon = /opt/csw/bin/dbus-daemon
+
```

Comment:
The linter doesn’t like this added newline ;)

Reply:
Done